be/src/olap/rowset/segment_v2/encoding_info.h

```diff
@@ -22,18 +22,16 @@
 #include "common/status.h"
 #include "gen_cpp/segment_v2.pb.h"
 #include "olap/types.h"
+#include "olap/rowset/segment_v2/options.h"
+#include "olap/rowset/segment_v2/page_builder.h"
+#include "olap/rowset/segment_v2/page_decoder.h"
 
 namespace doris {
 
 class TypeInfo;
 
 namespace segment_v2 {
 
-class PageBuilder;
```

Comment:
why change forward declaration to include?

Reply:
I will fall back to forward declaration